core/auth/fake/module.go

```diff
@@ -0,0 +1,56 @@
+package fake
+
+import (
+	"flamingo.me/dingo"
+
+	"flamingo.me/flamingo/v3/core/auth"
+	"flamingo.me/flamingo/v3/framework/config"
+)
+
+// Module provides Fake OpenID Connect support
```

Comment:
Please remove the text about OpenID Connect unless this package provides OpenID connect support

Reply:
done